docs/embed-c.md

```diff
@@ -1,4 +1,112 @@
 # Embedding Wasmtime in C
 
-... more coming soon
+This document shows an example of how to embed Wasmtime using the [C API](https://github.com/WebAssembly/wasm-c-api) to execute a simple wasm program.
 
+## Hello, world!
+
+An excellent example of C API usage can be found at https://github.com/WebAssembly/wasm-c-api/blob/master/example/hello.c
```

Comment:
I think it might be best to inline all the code here in this document rather than linking externally, because this external link could change or otherwise go away.